src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,54 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_ELEMENTS = 10;
+    private Object[] keys;
+    private Object[] values;
```

Comment:
```suggestion
    private Object[] keys;
    private Object[] values;
```
You can here already make your arrays type K and V  and not casting then the result of get method